spec/02-integration/02-cmd/15-utils_spec.lua

```diff
@@ -0,0 +1,72 @@
+describe("kong cli utils", function()
+
+  describe("nginx_signals", function()
+    local signals = require "kong.cmd.utils.nginx_signals"
+    local pl_path = require "pl.path"
+    local pl_file = require "pl.file"
+    local pl_dir = require "pl.dir"
```

Comment:
Minor: could we move these requires to the top of the file? We don't usually put the requires inside functions unless necessary.